.github/workflows/release.yml

```diff
@@ -1,28 +1,22 @@
 name: Make release
 
 on:
-  push:
-    tags:
-    - '*'
+  push: {}
 
-jobs:
-  release:
+# on:
```

Comment:
I assume this was just for the testing?
Only tagged versions should be released.

Reply:
I could have sworn I reverted that but yes it was for testing purposes on my fork. Fixed now! 👍